NEXT_RELEASE.md

```diff
@@ -0,0 +1,370 @@
+# Next Major Release - Developer Guide
+
+This guide explains how to contribute breaking changes to the next major version of the Polymesh SDK using the ephemeral integration system.
+
+## 📋 Overview
+
+Breaking changes for the next major release are managed through special branches that follow a specific naming convention. These branches are automatically discovered, validated, and integrated to create preview releases.
+
+## 🌿 Branch Naming Convention
+
+### Format
+```
+bc-<number>-<description>
+```
+
+### Rules
+- **Must start with `bc-`**
+- **Include a number** (determines integration order)
+- **Use lowercase letters, numbers, and hyphens only**
+- **Be descriptive** about the breaking change
+
+### Examples
+```bash
+bc-1-remove-deprecated-apis     # Removes old API methods
+bc-2-async-interface-changes    # Makes interfaces asynchronous
+bc-5-new-error-handling         # Changes error handling approach
+bc-10-update-type-definitions   # Updates TypeScript definitions
+```
+
+### Numbering Guidelines
+- **Start from 1** for the first breaking change
+- **Use sequential numbers** for logical ordering
+- **Numbers determine integration order**: bc-1 → bc-2 → bc-5 → bc-10
+- **Avoid conflicts**: Check for existing branches with the same number
+- **Keep under 50** (higher numbers indicate too many breaking changes)
+
+## 🚀 Creating a Breaking Change Branch
+
+### Step 1: Create the Branch
+```bash
+# Start from the latest develop
+git checkout develop
+git pull origin develop
+
+# Create your breaking change branch
+git checkout -b bc-3-remove-legacy-widget develop
+```
+
+### Step 2: Make Your Changes
+Implement your breaking changes in the branch:
+```bash
+# Make your changes
+# Edit files, remove deprecated code, etc.
+
+# Stage changes
+git add .
+```
+
+### Step 3: Commit with Proper Format
+Use conventional commits with breaking change indicators:
+```bash
+git commit -m "feat!: remove legacy Widget class
+
+BREAKING CHANGE: Removed the deprecated Widget.legacy() method
+
+Breaking-Change-Category: API
+Breaking-Change-Impact: High
+Breaking-Change-Migration: Replace Widget.legacy() with Widget.current()
+Breaking-Change-Affects: All applications using Widget.legacy()
+
+The old Widget.legacy() method has been removed as part of the v3.0
+modernization. Applications should migrate to Widget.current() which
+provides the same functionality with improved performance."
+```
+
+### Step 4: Push the Branch
+```bash
+git push origin bc-3-remove-legacy-widget
+```
+
+## 📝 Commit Message Format
+
+### Basic Structure
+```
+<type>!: <description>
+
+BREAKING CHANGE: <summary>
+
+Breaking-Change-Category: <category>
+Breaking-Change-Impact: <impact-level>
+Breaking-Change-Migration: <migration-instructions>
+Breaking-Change-Affects: <affected-components>
```

Comment:
I think commit citizen can be configured with a template. Automation is preferable to documentation